ERCS/erc-7573.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 7573
+title: Conditional-upon-Transfer-Decryption for DvP
+description: A Proposal for a Lean and Functional Delivery versus Payment
+author: Christian Fries (@cfries), Peter Kohl-Landgraf (@pekola)
+discussions-to: https://ethereum-magicians.org/t/erc-7573-conditional-upon-transfer-decryption-for-delivery-versus-payment/17232
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-12-05
+---
+
+## Abstract
+
+The interfaces model the functional transaction scheme to establish a secure delivery-versus-payment
```

Comment:
```suggestion
The interfaces in this proposal model a functional transaction scheme to establish a secure delivery-versus-payment
```

Reply:
Thank you. I have added some more details on this, linking to the literature via a csl-json.